bin/foreman-proxy-certs-generate

```diff
@@ -20,20 +20,33 @@ class ForemanProxyCertsGenerate
 
   private
 
+  # rubocop: disable Metrics/MethodLength
   def generate_answers_file
-    temp_file('foreman-proxy-certs-answer',
-              <<-EOF
+    if File.file?('/etc/foreman-installer/scenarios.d/last_scenario.yaml')
```

Comment:
@stbenjam ack will do on changing things for the better. Is this how the packaging pr should look like? 

sed -ri 'sX\./configX%{_sysconfdir}/foreman-installer/scenarios.dXg' config/foreman-proxy-certs-generate